models/reviews.model.js

```diff
@@ -15,3 +15,22 @@ exports.selectReviewById = (id) => {
       }
     });
 };
+
+exports.updateReviewById = (id, inc_votes) => {
+  return db
+    .query(
+      "UPDATE reviews SET votes = votes + $1 WHERE review_id = $2 RETURNING *",
+      [inc_votes, id]
+    )
+    .then(({ rows }) => {
+      if (rows.length === 0) {
+        return Promise.reject({
+          status: 404,
+          message: "Review ID does not exist",
+        });
+      } else {
+        rows[0].created_at = rows[0].created_at.toString();
```

Comment:
you don't really need to modify created at, because you're only patching, so the original creation date is valid, no?
You only need do this for a 'last modified' kinda key